lib/tasks/safe_reset.rake

```diff
@@ -11,6 +11,7 @@ namespace :db do
                    .map { |row| row["tablename"] }
       tables.delete "schema_migrations"
       tables.delete "spatial_ref_sys"
+      tables.delete "features"
```

Comment:
Do we actually want this? I would have thought we wanted the flags to persist between deploys on dev?